feature/subscriber-attributes/src/main/java/com/revenuecat/purchases/subscriberattributes/AttributionDataMigrator.kt

```diff
@@ -76,7 +76,7 @@ class AttributionDataMigrator {
 
     private fun convertMParticleAttribution(data: JSONObject): Map<String, String> {
         val mapping: Map<Any, String> = mapOf(
-            (AttributionKeys.MParticle.ID or AttributionKeys.NETWORK_ID) to SPECIAL_KEY_MPARTICLE_ID,
+            (AttributionKeys.NETWORK_ID or AttributionKeys.MParticle.ID) to SPECIAL_KEY_MPARTICLE_ID,
```

Comment:
guess i missed this before, but how does that give "preference" to network_id? never used a pair as a key to a map before...

Reply:
That `or` function just maps to `to` https://github.com/RevenueCat/purchases-android/pull/341/files/5b7610b5b9b763e56a901b41a603b6e84aba2d30#diff-0e844363d1d178f15c2db334e816055f346da3d3e8fe69dd8a0bb209efa1b2d9R147 for syntax sugaring. Then in line https://github.com/RevenueCat/purchases-android/pull/341/files/5b7610b5b9b763e56a901b41a603b6e84aba2d30#diff-0e844363d1d178f15c2db334e816055f346da3d3e8fe69dd8a0bb209efa1b2d9R140 I choose the first part of the pair over the second one.

Reply:
ah, couldn't find that place you used them. thanks